starter-code/functions-and-arrays.js

```diff
@@ -1,14 +1,18 @@
 // Find the maximum
 function maxOfTwoNumbers (first, second) {
-
+  return (first > second) ? first : second;
 }
 
 var largest = maxOfTwoNumbers(2, 6);
 console.log(largest);
 
 // Finding Longest Word
 function findLongestWord (words) {
-
+  var longest = words[0]
+  words.forEach(function(word){
+    longest = (word.length > longest.length) ? word : longest
```

Comment:
in this case the ternary is not necessary because we don't need the `else` statement